uint32 saliency_id
time ts
Float32XY position
float32 confidence
